README.md

```diff
@@ -108,4 +116,33 @@ check balance (you need to wait for your node catching up with the network)
 # then input y for the pop up to confirm send
 ```
 
+#### 4. run node as a service
+**NOTE:** All below steps require *root* privileges
 
+create file ```/lib/systemd/system/stratos.service``` with following content:
+```
+[Unit]
+Description=Stratos Node
```

Comment:
it's better to name as "Stratos Chain Node", since we need have storage node soon.